onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -87,24 +87,27 @@ void BasicBackend::StartAsyncInference(Ort::CustomOpApi& ort,
     auto graph_input_buffer = graph_input_blob->buffer()
                                   .as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type*>();
     size_t input_data_size = graph_input_blob->byteSize();
-
-    auto tensor_shape = ort.GetTensorTypeAndShape(input_tensors[i]);
+    
+    std::string input_name = input_info_iter->first;
+    const OrtValue* tensor = ort.KernelContext_GetInput(context, subgraph_context_.input_names.at(input_name));
```

Comment:
Shouldn't these changes be added to vad-m backend as well

Reply:
done